### ISSUE TITLE
Setting Bluebird Config breaks in projects where Bluebird is also used

### ISSUE THREAD

**Issue by mjreich** [OptimalBits/bull#1130]
## Description
If a module is loaded in a project before Bull that also uses Bluebird, bull with cause Bluebird to throw an exception by trying to set Bluebird.config.longStackTraces.

This really seems like debug code which shouldn't be in the release, or at minimum be set by a config option but default to unset.  Setting longStackTraces in the lib will only work when bull is the first module loaded before any others that also use bluebird.

```
Error: cannot enable long stack traces after promises have been created
 
  See http://goo.gl/MqrFmX

  at Function.Promise.longStackTraces (node_modules/bluebird/js/release/debuggability.js:121:15)
  at Function.Promise.config (node_modules/bluebird/js/release/debuggability.js:264:21)
  at Object.<anonymous> (node_modules/bull/lib/queue.js:44:10)
```

## Minimal, Working Test code to reproduce the issue.

```
var Bluebird = require('Bluebird')

const myPromise = new Promise((resolve, reject) => {...})

var bull = require('bull')
```

## Bull version
3.5.1

## Additional information
This isn't an issue with previous 3.3 versions, but haven't looked at more recent relaeses.

**Comment by alolis** [comment]
Maybe PR #1127 is related?

**Comment by manast** [comment]
Fixed in version 3.5.2.
